docs/CodeDesign.md

```diff
@@ -1,20 +1,23 @@
 # Requirements
 
-## Prototype
+## Initial goals
 
-The goal of the first prototype is to reach a working and usable tool with minimal functionality.
-Namely, a user would be able to run `fawltydeps` as a command line tool in their Python project.
-FawltyDeps will:
+We want a working and usable tool that slowly grows functionality as needed.
+A user would be able to run `fawltydeps` as a command line tool in their Python
```

Comment:
I think it makes sense and reads more naturally if we match verb tense across these items. I don't have a strong preference about _which_ tense, just agreement among them

Reply:
```suggestion
A user runs `fawltydeps` as a command line tool in their Python
```